apps/console/src/features/groups/components/group-common-components/render-chip.tsx

```diff
@@ -84,12 +85,15 @@ export const RenderChipRolesInGroups: FunctionComponent<RenderChipInterface> = (
             <Chip
                 { ...props }
                 key={ key }
-                label={ 
+                label={
                     (<>
-                        <i> { audienceType } </i> 
-                        <i> { audienceType === "application" && ( " : " + audienceDisplay ) } </i>
-                        { " | " }
-                        <strong> { displayName } </strong>
+                        <b >{ displayName }</b>
```

Comment:
Shall we use the `Typography` component instead of these native HTML elements.
It has `fontWeight` props etc.

Reply:
For `italic` check if there's a prop or else use the `sx` property.

Reply:
Not a blocker for the PR merge. @dasuni-30

Reply:
Improved with 06add04b94407268aa58f36abbf331e8e68ac266.